internal/forc/lib/install.js

```diff
@@ -69,5 +79,6 @@ import {
 
     // Cleanup
     rmSync(pkgPath);
+    rmSync(madeFromGitPath, { force: true });
```

Comment:
You should keep the built repo in place so you can re-use it when you switch back to it.

Reply:
The `madeFromGitPath` naming might have been confusing. I refactored this. All in all, the binaries built by `cargo build --release` are untouched and will be reused in the `buildFromGitBranch` flow.